src/components/dls/QuranWord/QuranWord.tsx

```diff
@@ -85,8 +85,10 @@ const QuranWord = ({
     wordText = <GlyphWord font={font} text={getGlyph(word, font)} pageNumber={word.pageNumber} />;
   } else if (font === QuranFont.Tajweed) {
     wordText = <TajweedWord path={word.text} alt={word.textUthmani} />;
-  } else if (word.charTypeName !== CharType.Pause) {
-    wordText = <TextWord font={font} text={word.text} charType={word.charTypeName} />;
+  } else {
+    wordText = (
```

Comment:
is `CharType.Pause` handled in this case?

Reply:
We don't need `CharType.Pause` anymore. It is removed from the BE, pause marks are already merged with their previous word.

We actually can remove `word.charTypeName`, we now have only `word` and `end`( ayah number) char types. And sending `char_type_name` attribute for each word is bit overkill IMO. 

We can just add `isEnd` to `end` words, just like we're sending `highlighted` to words that should be highlighted. Let me know if you've a better solution @osamasayed